cmd/ocsp-responder/main.go

```diff
@@ -149,12 +157,17 @@ func makeDBSource(dbMap dbSelector, issuerCert string, reqSerialPrefixes []strin
 	if err != nil {
 		return nil, fmt.Errorf("Could not parse issuer cert %s: %s", issuerCert, err)
 	}
-	if len(caCert.SubjectKeyId) == 0 {
-		return nil, fmt.Errorf("Empty subjectKeyID")
+	var pkixPublicKey struct {
+		Algo      pkix.AlgorithmIdentifier
+		BitString asn1.BitString
+	}
+	if _, err := asn1.Unmarshal(caCert.RawSubjectPublicKeyInfo, &pkixPublicKey); err != nil {
```

Comment:
I think [MarshalPKIXPublicKey](https://godoc.org/crypto/x509#MarshalPKIXPublicKey) does this for us, right?

Reply:
MarshalPKIXPublicKey generates an encoded SPKI structure.

Reply:
(I think you actually have a golang ticket pointing this out, since there is no such thing as a 'PKIX public key')

Reply:
Oh right, where this isn't the SPKI field itself, but the inner guts, per https://tools.ietf.org/html/rfc6960#page-13:

   o  issuerKeyHash is the hash of the issuer's public key.  The hash
      shall be calculated over the value (excluding tag and length) of
      the subject public key field in the issuer's certificate.

Can you add a comment here with a quote and link to the spec, plus an explanation of why MarshalPKIXPublicKey doesn't do it for us?

Also, please add a test.